java/vector/src/test/java/org/apache/arrow/vector/util/TestVectorAppender.java

```diff
@@ -95,6 +96,27 @@ public void testAppendFixedWidthVector() {
     }
   }
 
+  @Test
+  public void testAppendEmptyFixedWidthVector() {
+    try (IntVector target = new IntVector("", allocator);
+         IntVector delta = new IntVector("", allocator)) {
+
+      target.allocateNew(10);
```

Comment:
This is not needed, as `ValueVectorDataPopulator.setVector` will do the allocation.

Reply:
I removed all explicit calls for allocation if the utility method performs the allocation.